ui_framework/src/components/table/_table.scss

```diff
@@ -0,0 +1,70 @@
+/**
+ * NOTE: table-layout: fixed causes a bug in IE11 and Edge (see #9929). It also prevents us from
+ * specifying a column width, e.g. the checkbox column.
+ */
+.kuiTable {
+  width: 100%;
+  border: none;
+  border-collapse: collapse;
+  font-size: $kuiTableFontSize;
+}
+
+.kuiTableHeaderCell {
+  @include kuiTableCell;
+
+  padding: $kuiTableCellPadding;
+  color: $kuiTitleColor;
+}
+
+.kuiTableHeaderCell--alignRight {
+  text-align: right;
+}
+
+  .kuiTableHeaderCell__liner {
+    display: inline-block;
+  }
+
+.kuiTableRow {
+  &:hover {
+    background-color: rgba($kuiColorPrimary, 0.05);
+  }
+
+  &.kuiTableRow-isSelected {
+    background-color: rgba($kuiColorHighlight, 0.05);
+  }
+}
+
+.kuiTableRowCell {
+  @include kuiTableCell;
+
+  color: $kuiTextColor;
+  border-top: $kuiBorderThin;
```

Comment:
It's on border-bottom as well (right now it doesn't exist on the last row). You'll want to collapse the border on your table.

![image](https://user-images.githubusercontent.com/324519/28842828-07494722-76b4-11e7-8f07-1344126fa6ed.png)